src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,23 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  (comments.length > 0)
+    ? (
+      <div className="CommentList">
+        {comments.map(comment => (
+          <CommentInfo
+            comment={comment}
+            key={comment.id}
+          />
+        ))}
+      </div>
+    )
+    : (
+      <>
+        <hr />
```

Comment:
Avoid using `<hr>` tags for styling. It's better to use CSS for styling. You might want to create a class that adds a border or line in your CSS.